job_spec/AUTORIFT_ITS_LIVE_TEST.yml

```diff
@@ -0,0 +1,59 @@
+AUTORIFT_TEST:
+  required_parameters:
+    - granules
+  parameters:
+    granules:
+      default:  '""'
+      api_schema:
+        type: array
+        minItems: 2
+        maxItems: 2
+        example:
+          - S2B_13CES_20200105_0_L1C
+          - S2B_13CES_20200315_0_L1C
+        items:
+          anyOf:
+            - description: The name of the Sentinel-1 SLC granule to process
+              type: string
+              pattern: "^S1[AB]_IW_SLC__1S[SD][VH]"
+              minLength: 67
+              maxLength: 67
+              example: S1A_IW_SLC__1SSV_20150621T120220_20150621T120232_006471_008934_72D8
+            - description: The name of the Sentinel-2 granule to process (ESA naming convention)
+              type: string
+              pattern: "^S2[AB]_MSIL1C_"
+              minLength: 60
+              maxLength: 60
+              example: S2A_MSIL1C_20200627T150921_N0209_R025_T22WEB_20200627T170912
+            - description: The name of the Sentinel-2 granule to process (Element 84 Earth Search naming convention)
+              type: string
+              pattern: "^S2[AB]_.*_L1C"
+              minLength: 23
+              maxLength: 25
+              example: S2A_22WEB_20200627_0_L1C
+            - description: The name of the Landsat 4, 5, 7, 8 or 9 Collection 2 granule to process
+              type: string
+              pattern: "^L([CO]0[89]|E07|T0[45])_L1"
+              minLength: 40
+              maxLength: 40
+              example: LC08_L1GT_118112_20210107_20210107_02_T2
+    bucket_prefix:
+      default:  '""'
+  validators: []
+  tasks:
+    - name: ''
+      image: ghcr.io/asfhyp3/hyp3-autorift
+      image_tag: test
+      command:
+        - --bucket
+        - '!Ref Bucket'
+        - --bucket-prefix
+        - Ref::bucket_prefix
+        - --parameter-file
+        - '/vsicurl/http://its-live-data.s3.amazonaws.com/autorift_parameters/v001/autorift_landice_0120m.shp'
+        - --naming-scheme
+        - ITS_LIVE_PROD
```

Comment:
:thinking: for test jobs, I wonder if we should keep the default ASF OD naming scheme to not confuse them with production jobs...

I guess it's a better test to use the same naming scheme here and makes golden testing easier as well.